javascript/package.json

```diff
@@ -165,7 +162,7 @@
     "ng-packagr": "16.1.0",
     "node-fetch": "^2.6.7",
     "nodemailer": "^6.7.7",
-    "nx": "16.5.1",
+    "nx": "16.7.4",
```

Comment:
Have you upgraded nx manually? Angular should have been updated to 16.2.3
![2023-09-01_07-56](https://github.com/taigaio/taiga/assets/905195/06bd98f6-8aed-4120-ab88-464561b9ad67)

Reply:
I follow the steps from the updagrade.workflow.md but I can apply them again just in case